### PR TITLE
updated leave on buying more vacation

### PR DESCRIPTION
**Pull request by demkodo**
I updated the leave section on buying more vacation days or deduct the days from next year's quota. Also included the link to the folder on Drive with the form you need to sign to approve deducting the extra days from your salary as well as how to calculate what extra time off actually costs based on your salary.

-----
[View rendered activities/staff-management/leave.md](https://github.com/publiccodenet/about/blob/update-staff-leave/activities/staff-management/leave.md)

### CODE REVIEW

File: activities/staff-management/leave.md [hunk 1]
```diff
@@ -31,6 +31,14 @@ More days can be bought at cost rate with agreement from your manager and <hr@pu
 
 The leave can now be approved by your manager.
 
+### Buying more vacation time or deduct from next year's quota
+
+Firstly, you can see how much leave you have when logging into your Tentoo NMBRS account. Navigate to the "leave" section on your dashboard.
+
+Please let the Operations Coordinator know if you want to take more vacation time than you have an if you want the additional leave to be deducted from next year's quota (Tentoo NMBRS does this automatically) or if you want to buy more days.
+
+When buying more vacaction time you need to fill out a form approving the deduction from your salary. You may also want to know what extra vacation time actually costs. Both the form as well as how to calculate the costs [can be found on the Drive.](https://drive.google.com/drive/folders/1rZqzT6FTnHzMLRbkdiM5DPz3Hie5TGZc)
```
Comment (ElenaFdR) [inline activities/staff-management/leave.md:40]:
```suggestion
When buying more vacaction time you need to fill out a form approving the deduction from your salary. You may also want to know what extra vacation time actually costs. Both the [form as well as how to calculate the costs](https://drive.google.com/drive/folders/1rZqzT6FTnHzMLRbkdiM5DPz3Hie5TGZc) are on Google Drive.
```

**Comment by Ainali** [comment]
This is good, but does both files need to be in Drive? The template I understand since it's made for print, but the other one could perhaps be a page on About?

**Comment by demkodo** [comment]
> This is good, but does both files need to be in Drive? The template I understand since it's made for print, but the other one could perhaps be a page on About?

Hi Jan, with "the other one" you mean the file with the calculations?

**Comment by Ainali** [comment]
> Hi Jan, with "the other one" you mean the file with the calculations?

Yes. At least I can see only two files in the linked folder, and I mean the one that is not the letter template.

**Comment by demkodo** [comment]
> > This is good, but does both files need to be in Drive? The template I understand since it's made for print, but the other one could perhaps be a page on About?
> 
> Hi Jan, with "the other one" you mean the file with the calculations?



> > Hi Jan, with "the other one" you mean the file with the calculations?
> 
> Yes. At least I can see only two files in the linked folder, and I mean the one that is not the letter template.

I will update it and point to the Drive for the form and add the calculations inline on the page itself - thanks!

**Review by bvhme — APPROVED** [review]
Looks good.

**Pull request merged**